### ISSUE TITLE
Type imports

### ISSUE THREAD

**Issue by aboeglin** [madlib-lang/madlib#46]
Proposed syntaxes:

```madlib
import type { Maybe } from "Maybe"

import { type Maybe, Just, Nothing } from "Maybe"
```

We then also need an export type that would look like this:

```madlib
type Maybe a = Just a | Nothing 

// export the type only
export type Maybe

// export constructors
export Just
export Nothing
```

**Comment by aboeglin** [comment]
So regarding imports, after a discussion on slack we decided to only include the second import syntax for now. Precisely:

```madlib
import { type Maybe, Just, Nothing } from "Maybe"
```

**Comment by aboeglin** [comment]
Closed by #47
